packages/e2e-tests/lib/Tar.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+var __importDefault = (this && this.__importDefault) || function (mod) {
+    return (mod && mod.__esModule) ? mod : { "default": mod };
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.Tar = void 0;
+const utilities_1 = require("./utilities");
+const node_child_process_1 = __importDefault(require("node:child_process"));
+const node_fs_1 = __importDefault(require("node:fs"));
+class Tar {
```

Comment:
assuming this is third party code as well?
maybe the source for these clsses in lib can be documented?

Reply:
same as above